From 8b4fd18c98eab1287836c79ae4efc77541e7dc30 Mon Sep 17 00:00:00 2001
From: Alon Bar-Lev <alon.barlev@gmail.com>
Date: Wed, 18 Apr 2018 23:53:01 +0300
Subject: [PATCH GPGME] build: gpgme-json: install properly

not installing properly using libtool result in:

 * QA Notice: The following files contain insecure RUNPATHs
 *  Please file a bug about this at https://bugs.gentoo.org/
 *  with the maintainer of the package.
 *   /var/tmp/portage/app-crypt/gpgme-1.11.0/image/usr/bin/gpgme-json
 *     RPATH: /var/tmp/portage/app-crypt/gpgme-1.11.0/work/b/src/.libs
---
 src/Makefile.am | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index c2d4a843..3d638b23 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -103,9 +103,7 @@ gpgme_tool_SOURCES = gpgme-tool.c argparse.c argparse.h
 gpgme_tool_LDADD = libgpgme.la @LIBASSUAN_LIBS@
 
 gpgme_json_SOURCES = gpgme-json.c cJSON.c cJSON.h
-gpgme_json_LDADD = -lm libgpgme.la $(GPG_ERROR_LIBS)
-# We use -no-install temporary during development.
-gpgme_json_LDFLAGS = -no-install
+gpgme_json_LDADD = -lm libgpgme.la @GPG_ERROR_LIBS@
 
 
 if HAVE_W32_SYSTEM
-- 
2.16.1

